src/main/scala/com/scylladb/migrator/readers/Cassandra.scala

```diff
@@ -126,7 +128,16 @@ object Cassandra {
 
       timestampsToFields
         .map {
-          case ((ttl, writetime), fields) =>
+          case ((ttl, writetime), fields) => {
+            val writetimestamp = writetime.getOrElse(CassandraOption.Unset)
+            val baseTTL = if (defaultTTL > 0) defaultTTL else 0L
+            val deltaTTL = if (writetimestamp == CassandraOption.Unset) {
+              0
+            } else {
+              baseTTL - (System.currentTimeMillis - writetimestamp.asInstanceOf[Long] / 1000) / 1000
+            }
+            val finalttl =
+              if (writetimestamp != CassandraOption.Unset && (deltaTTL > 0)) deltaTTL else 0L
```

Comment:
This is written off hand, and likely not compiling, though surely there's something akin to `abs` for a `Long` in Scala. :)
```suggestion
            val finalttl = if (writetimestamp == CassandraOption.Unset) {
              0
            } else {
              (defaultTTL.abs() - (System.currentTimeMillis - writetimestamp.asInstanceOf[Long] / 1000) / 1000).abs()
            }
```

Reply:
I cannot do abs
I want to avoid negative defaultTTL from config file
and negative TTL from diff

actually negative TTL from diff would mean this record is already passed default TTL and should expire asap (so ideally it shouldn't even be written)